examples/https-termination/README.md

```diff
@@ -1,14 +1,14 @@
 # HTTPS Termination Example
 
 In this example, we expand on the simple [cafe-example](../cafe-example) by adding HTTPS termination to our routes and
-an HTTPS redirect from port 80 to 443. We will also show how you can use a ReferenceGrant to permit your Gateway to
-reference a Secret in a different Namespace.
+an HTTPS redirect from port 80 to 443. We will also show how you can use a `ReferenceGrant` to permit your `Gateway` to
+reference a `Secret` in a different `Namespace`.
```

Comment:
Namespace isn't formatted like this in the previous examples. I think we should pick one approach and stick with it. My vote is we don't use backticks for Kubernetes Resources.